README.md

```diff
@@ -1,40 +1,36 @@
+# app-chyrondetection
 
-___
-# Parameters
-This CLAMS app accepts the following query parameters
+## User instruction
 
-- pretty
-- sampleRatio: integer, default: 10
+General user instruction for CLAMS apps is available at [CLAMS Apps documentation](https://apps.clams.ai/clamsapp/).
 
----
+We provide a `Containerfile`. If you want to run this app as a docker container (not worrying about dependencies), build 
+an image from the `Containerfile` via the following command in the app directory:
 
-### Using docker
-We provide a [`Dockerfile`](Dockerfile). If you want to run the slate detector as a docker container (not worrying about dependencies), build an image from the `Dockerfile` and run it with the target directory mounted to `/data`. Just MAKE SURE that target directory is writable by others (`chmod o+w $TARGET_DIR`). For example, 
+```bash
+docker build . -f Containerfile -t chyrons
+```
+
+Then, run the image with the target directory mounted to `/data`. Just MAKE SURE that target directory is writable by others (`chmod o+w $TARGET_DIR`):
+
+```bash
+docker run -v path/to/target/data:/data -p 5000:5000
+```
+
+From here, the app can be run through the terminal, by posting a mmif file to the host port:
+
+```bash
+curl -X POST -d @path/to/target/mmif http://0.0.0.0:5000
+```
+
```

Comment:
Not sure these instructions are necessary here, as most of them (building container, running container, mounting, POST-ing with params) are in the 'generic' manual linked at the top of this file. 

One thing I wonder is why `/data` directory has to be writable, though. That's is a very unusual requirement for a CLAMS app.

Reply:
the write permissions in `/data` along with most of the formatting of the instructions in this section come from the README for another one of the apps (I've been trying to make the information as consistent as possible across all the apps for UX purposes). I copied it over relatively uncritically, but since this entire section is redundant then we can get rid of it.